frontend_lib/src/css/Variable.styl

```diff
@@ -27,13 +27,24 @@ zIndexOverHeader = 50
 headerHeight = 61px // INFO - CH - 2019-04-19 - the total header height. Used to calculate height of layout like height: "calc(100% - %s)" % headerHeight
 sidebarWidth = 300px
 
-fontSizeTitle = 25px
-fontSizeMedium = 20px
-fontSizeSmall = 14px
+fontStandard = 16px
+mainTitle = 25px
+mediumTitle = 20px
+smallTitle = 14px
 listItemHeight = 5px
 popupWidth = 420px
 bigIconSize = 30px
 
+/*************************/
+/**** SPACING ****/
+
+itemSpacing = 5px
+standardSpacing = 10px
+borderSpacing = 15px
+containerSpacing = 20px
+blocSpacing = 30px
+pageSpacing = 50px
```

Comment:
We need all these spaces? I can see the page one, for the rest, can't we use the standard?

Reply:
```suggestion
textSpacing = 5px
standardSpacing = 10px
pageSpacing = 30px
```

Reply:
We should also add a pageSpacing around the publications, and use standartdSpacing between each publication. 

In order to add a 30px margin at the bottom of each tabBar: 
`
.tabBar
  margin-bottom pageSpacing
 & .pageTitleGeneric
  margin 0
`